insights/specs/__init__.py

```diff
@@ -231,6 +231,7 @@ class Specs(SpecSet):
     nfs_exports_d = RegistryPoint(multi_output=True)
     nfs_exports = RegistryPoint()
     nginx_conf = RegistryPoint(multi_output=True)
+    nmcli_dev_show = RegistryPoint()
```

Comment:
@vishwanathjadhav you also need to add `nmcli_dev_show` to `insights/specs/defaults.py` as a `simple_command`.

Reply:
@vishwanathjadhav - and please add its corresponding "simple_file" to `insights/specs/insights_archive.py` and `insights/specs/sos_archive.py` (if necessary)